help/C/Help_ch_Transactions.xml

```diff
@@ -43,8 +43,9 @@ Translators:
     </listitem>
 
     <listitem>
-      <para>Importing transactions either from a <acronym>QIF</acronym>/<acronym>OFX</acronym> file or via
-      "<acronym>OFX</acronym> on-line banking".</para>
+      <para>Importing transactions either from an <acronym>OFX</acronym>,
+        <acronym>QFX</acronym>, <acronym>QIF</acronym> or <acronym>CSV</acronym>
+        file or via "<acronym>OFX</acronym> on-line banking".</para>
     </listitem>
```

Comment:
I would prefer this spot not to delineate ANY of the formats; it would be appropriate here to say:
"Importing transactions either from a file or via on-line banking."
The fact that there isn't good documentation on importing is not a reason to add these formats here. It seems to me that the Guide is a better place to put information about importing data, since importing involves so much more than simply pressing buttons and choosing options.